2/scss/header.scss

```diff
@@ -0,0 +1,372 @@
+    max-width: 670px;
+    padding-left: 15px;
+    border-left: 1px solid $colorTextPrimary;
+    filter: drop-shadow(0px 2px 6px rgba(26, 26, 26, 0.3));
+  }
+  .bottom-button-why-skout {
+    margin: 20px auto 0;
+    .btn {
+      @include button-primary-first(130px,30px,$colorTextlinks,$colorTextPrimary);
+      border: 0;
+    }
+    .btn:hover {
+      @include button-hover-shadow;
+      position: relative;
+      bottom: 3px;
+    }
+  }
+}
+.is-hide {
+  overflow: hidden;
+}
+
+@media (min-width: 768px) {
+  .header {
+    .container {
+      position: relative;
+      padding: 28px 54px 50px;
+    }
+  }
+  .logo {
+    font-size: 24px;
+  }
+  .container-header-menu {
+    top: 0;
+    z-index: 2;
+    left: 50%;
+    .header-menu {
+      margin-top: 70px;
+      &-item {
+        margin-bottom: 0;
+      }
+      .active-link {
+        &::after {
+          left: 32px;
+        }
+      }
+    }
+  }
+  .header-text {
+    max-width: 1100px;
+    margin: 80px auto;
+    &-value {
+      font-size: 70px;
+      line-height: 84px;
+    }
+  }
+  .bottom-text-container {
+    flex-direction: row;
+    .bottom-text {
+      @include text-size-small;
+      padding: 0 30px;
+    }
+    .bottom-button-why-skout {
+      margin: auto;
+      .btn {
+        min-width: 200px;
+        min-height: 40px;
+      }
+    }
+  }
+  .hamburger-icon.open-menu {
+    z-index: 3;
+    &::before{
+      background-color: $colorTextlinks;
+    }
+    &::after{
+      background-color: $colorTextlinks;
+    }
+  }
+  .sticky-main {
+    background-color: $colorTextPrimary;
+    transition: background-color 600ms;
+    max-width: 1440px;
+    padding: 29px 90px;
+    z-index: 1;
+    margin: auto;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    .logo {
+      color: $hoverColorLink;
+    }
+    .container-header-menu {
+      .header-menu {
+        &-item {
+          .link {
+            color: $hoverColorLink;
+            &:hover {
+              color: $colorTextlinks;
+            }
+          }
+        }
+      }
+      .enter-buttons {
+        button {
+          color: $hoverColorLink;
+          border: 1px solid $hoverColorLink;
+          &:hover {
+            color: $colorTextlinks;
+            border: 1px solid $colorTextlinks;
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (min-width: 1440px) {
+  .hamburger-icon {
+    display: none;
+    position: absolute;
+  }
+  .header {
+    .container {
+      min-height: 900px;
+      padding: 28px 90px 100px;
+    }
+    .added-empty-field {
+      padding: 78px 90px 100px;
+    }
+  }
+  .header-inner {
+    align-items: center;
+  }
+  .logo {
+    font-size: 28px;
+    margin-right: 90px;
+  }
+  .container-header-menu {
+    display: flex;
+    position: static;
+    background-color: $color-bg-transparent;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-between;
+    .header-menu {
```

Comment:
This will be compiled to `.container-header-menu .header-menu`

You have only one header menu. No need to write styles inside `.container-header-menu`

This happens everywhere in your css. Try not to do like this in the future.

Reply:
I understood